extension/src/prototypes/ui-components/OverlayPopper.tsx

```diff
@@ -82,7 +82,9 @@ export const OverlayPopper: FC<OverlayPopperProps> = ({
 }) => {
   const handleClickAway = useCallback(() => {
     if (!pinned) {
-      onClose?.();
+      setTimeout(() => {
+        onClose?.();
+      }, 0);
```

Comment:
Why is this timeout needed?

Reply:
It's for react-dnd. I added a comment.
[comment](https://github.com/eukarya-inc/PLATEAU-VIEW-3.0/pull/14/commits/7157f6e2495536be04267342e94af09daa918072)